R/prepare-lpr.R

```diff
@@ -81,33 +81,86 @@ prepare_lpr2 <- function(lpr_adm, lpr_diag) {
     )
 }
 
-#' Join together the LPR3 (`diagnoser` and `kontakter`) registers.
+#' Prepare and join the two LPR3 registers to extract diabetes and pregnancy diagnoses.
 #'
-#' To prepare the LPR3 data for the inclusion process, this function also
-#' renames the `cpr` variable to `pnr` to match the other registers.
+#' The output is used as inputs to `include_diabetes_diagnoses()` and to
+#' `get_pregnancy_dates()` (see exclusion events).
+#'
```

Comment:
```suggestion
#' @inherit prepare_lpr2 description
```

To reuse the description from `prepare_lpr2`